static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -418,7 +418,7 @@ const PublishPreview = ({ isRevision }) => {
           </Row>
         )
       }
-      <ErrorBoundary>
+      <ErrorBoundary previousURL={window.location.pathname}>
```

Comment:
since this is a global variable already, do we need to include it as a prop?

Reply:
yes because this capures the previousURL. If I were to call window.location.pathname in the file that this will be utilized, it would call the currentURL. I need this one so that the component that renders the error knows the user came from publishpreview